app/src/main/res/values/colors.xml

```diff
@@ -11,4 +11,6 @@
     <color name="dark_white">#E8E8E8</color>
     <color name="battleship_gray">#64707b</color>
     <color name="green">#8dbf75</color>
+
+    <color name="grey_222222">#222222</color>
```

Comment:
use color name instead of appending hex code

Reply:
if there are multiple combinations of grey, then how we will be using it?

Reply:
use the appropriate name of color according to the hex code